ivy_tests/test_ivy/helpers.py

```diff
@@ -873,26 +874,27 @@ def value_test(
             ret_np_flat, ret_np_from_gt_flat
         )
     )
-    # value tests, iterating through each array in the flattened returns
-    if not rtol:
-        for ret_np, ret_from_np in zip(ret_np_flat, ret_np_from_gt_flat):
-            rtol = TOLERANCE_DICT.get(str(ret_from_np.dtype), 1e-03)
-            assert_all_close(
-                ret_np,
-                ret_from_np,
-                rtol=rtol,
-                atol=atol,
-                ground_truth_backend=ground_truth_backend,
-            )
-    else:
-        for ret_np, ret_from_np in zip(ret_np_flat, ret_np_from_gt_flat):
-            assert_all_close(
-                ret_np,
-                ret_from_np,
-                rtol=rtol,
-                atol=atol,
-                ground_truth_backend=ground_truth_backend,
-            )
+    if test_values:
```

Comment:
this parameter seems a bit pointless, why would we ever call the function `value_test` with argument `test_values=False`?